CHANGELOG-LATEST.md

```diff
@@ -1,4 +1,5 @@
 # @financialforcedev/orizuru-transport-rabbitmq
 
-## Latest changes (not yet released)
```

Comment:
Minor, but this heading should not be removed, it stays in the CHANGELOG-LATEST to make it clear the changes are unreleased once the branch is merged into master.

Reply:
Change undone